contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcMeasureConstants.java

```diff
@@ -33,15 +33,47 @@
    * https://github.com/grpc/grpc/blob/master/doc/statuscodes.md.
    *
    * @since 0.8
+   * @deprecated in favor of {@link #RPC_CLIENT_STATUS} and {@link #RPC_SERVER_STATUS}.
    */
-  public static final TagKey RPC_STATUS = TagKey.create("canonical_status");
+  @Deprecated public static final TagKey RPC_STATUS = TagKey.create("canonical_status");
 
   /**
    * Tag key that represents a gRPC method.
    *
    * @since 0.8
+   * @deprecated in favor of {@link #RPC_CLIENT_METHOD} and {@link #RPC_SERVER_METHOD}.
    */
-  public static final TagKey RPC_METHOD = TagKey.create("method");
+  @Deprecated public static final TagKey RPC_METHOD = TagKey.create("method");
+
+  /**
+   * Tag key that represents a client gRPC canonical status. Refer to
+   * https://github.com/grpc/grpc/blob/master/doc/statuscodes.md.
+   *
+   * @since 0.13
+   */
+  public static final TagKey RPC_CLIENT_STATUS = TagKey.create("grpc_client_status");
```

Comment:
Why not GRPC_CLIENT_STATUS?

Reply:
Done. I updated all the new constants with prefix "GRPC_".